src/main/java/org/apereo/cas/configuration/model/core/CasServerProperties.java

```diff
@@ -0,0 +1,80 @@
+package org.apereo.cas.configuration.model.core;
+
+import io.cos.cas.osf.configuration.model.DevModeProperties;
+
+import org.apereo.cas.CasProtocolConstants;
+import org.apereo.cas.configuration.model.core.web.tomcat.CasEmbeddedApacheTomcatProperties;
+import org.apereo.cas.configuration.support.RequiredProperty;
+import org.apereo.cas.configuration.support.RequiresModule;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+
+import java.io.Serializable;
+
+/**
+ * This is {@link CasServerProperties}.
+ *
+ * <p>OSF CAS Customization: add {@link DevModeProperties devMode} to {@link CasServerProperties}; this allows
+ * dev mode options to be easily accessed from {@link org.apereo.cas.configuration.CasConfigurationProperties}
+ * by calling {@code casProperties.getServer().getDevMode()}.</p>
```

Comment:
For CR: needs to overlay class `CasServerProperties` (`cas.server` in settings) to add our `DevModeProperties` (`.dev-mode.*` in settings), e.g.:

```yaml
cas.server.dev-mode.allow-force-authn-exception=${ALLOW_FORCE_AUTHN_EXCEPTION:false}
cas.server.dev-mode.allow-force-http-error=${ALLOW_FORCE_HTTP_ERROR:false}
```